cfme/tests/control/test_actions.py

```diff
@@ -215,14 +215,16 @@ def action_collection(appliance):
     return appliance.collections.actions
 
 
-@pytest.fixture(scope="module")
+@pytest.yield_fixture(scope="module")
```

Comment:
this change is not necessary in pytest 3

Reply:
@izapolsk agree, but @mshriver insists to keep it.